libs/i18n-labels/en/datasets.json

```diff
@@ -2506,6 +2506,11 @@
       "vessel_id": "vessel_id"
     }
   },
+  "public-rfmo": {
```

Comment:
did you update this by hand? won't it be overwritten by the automated workflow of the datasets translations

Reply:
you are right, I'll remove them